2018-06-12 19:39:55 [WARN ](.p.r.c.DefaultPinpointClientHandler) DefaultPinpointClientHandler@57ea113a exceptionCaught() occurred. state:RUN_SIMPLEX. Caused:远程主机强迫关闭了一个现有的连接。
java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:108)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:337)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:89)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	at org.jboss.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.jboss.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
2018-06-12 19:39:58 [WARN ](.r.c.ConnectionFactory$ConnectEvent) try reconnect. connectAddress:DnsSocketAddressProvider{host='172.16.17.90', port=9996}
2018-06-12 19:39:59 [WARN ](.p.r.c.DefaultPinpointClientHandler) DefaultPinpointClientHandler@4b45dcb8 exceptionCaught() occurred. state:RUN_SIMPLEX. Caused:远程主机强迫关闭了一个现有的连接。
java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:108)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:337)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:89)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	at org.jboss.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.jboss.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
2018-06-12 19:40:01 [WARN ](.r.c.ConnectionFactory$ConnectEvent) try reconnect. connectAddress:DnsSocketAddressProvider{host='172.16.17.90', port=9996}
2018-06-12 19:40:01 [WARN ](c.n.p.r.c.DefaultPinpointClient    ) reconnectClientHandler:DefaultPinpointClientHandler@28218e7a{channel=[id: 0x89f914a4, /172.16.20.29:53970 => /172.16.17.90:9996]state=SocketState(NONE->BEING_CONNECT)}
2018-06-12 19:40:02 [WARN ](.r.c.ConnectionFactory$ConnectEvent) try reconnect. connectAddress:DnsSocketAddressProvider{host='172.16.17.90', port=9995}
2018-06-12 19:40:02 [WARN ](c.n.p.r.c.DefaultPinpointClient    ) reconnectClientHandler:DefaultPinpointClientHandler@756e3167{channel=[id: 0x4331c45a, /172.16.20.29:53972 => /172.16.17.90:9995]state=SocketState(NONE->BEING_CONNECT)}
